python/tests/test_data_array.py

```diff
@@ -54,33 +54,33 @@ def test_init_from_variable_views():
     var = sc.Variable(['x'], values=np.arange(5))
     a = sc.DataArray(data=var,
                      coords={'x': var},
-                     unaligned_coords={'meta': var},
+                     attrs={'meta': var},
```

Comment:
This is maybe a sign that what I mentioned on Slack about the name `meta` is not entirely wrong?
Intuitively, I think users may think that `meta` is just containing small bits of informationm such as the date of the experiment of the sample name, while in fact it also contains all the coordinates.

That said, I don't really have a better name than `meta` to designate both `coords` and `attrs`. The only one I can think of is `attrs` could be the collective term (a bit like python object attributes), and `meta` could be what we are currently calling `attrs`, but it doesn't feel great and would not resemble `xarray`'s attrs.

Reply:
Also, it's not entirely bad here, because `'meta'` will end up being part of `a.meta`...

Reply:
I had a similar thought @nvaytet about `meta` == `metadata` (auxiliary information only)  but I think it'll be simple to learn and remember the difference after encountering once, this is also reasonably well documented.

Reply:
For me coords *are* metadata: You make a measurement and get "296.6 Kelvin". That is the data. The metadata for this data point is *when* and *where* you measured it, which scipp calls coords.

Reply:
I view `meta/metadata` to be a good name here. From a data-science perspective the idea that the data is almost useless without the relevant metadata giving it context is fairly well established. This is why most modern data formats have metadata built into the file/storage solution. 

Whilst it may not be intuitive to the scientists currently, I feel as @OwenArnold says that it would be simple to learn, and is not out of step from a more data orientated view point.